docs/UserGuide.md

```diff
@@ -146,25 +146,68 @@ Examples:
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+
+:information_source: **Note:** Start time should be of format “dd-MM-yyyy HH:mm” (date-month-year Hour:minutes in 24 hr format).
+
+* `elist` followed by `eedit` 2 edits the 2nd event from the event list.
+* Edits the event at the specified `INDEX`. `INDEX` refers to the index number shown in the displayed event list. `INDEX` **must be a positive integer** eg 1, 2, 3, …​
+* **At least one** of the optional fields must be provided.
+* Existing values will be updated to the input values.
+* use `t/` to add a tag.
+* use `dt/` to remove a tag.
+* use `dt/*` to remove **all** tags
+* When editing tags, the tags to be deleted will be removed first, before new tags are added
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `elist` followed by `eedit 2 n/CS2103T Exam dt/Easy_exams t/Hard_exams` changes the name and tag of the 2nd event on the event list to `CS2103T Exam` and `Hard_exams` respectively.
+* `efind Betsy` followed by `eedit 2 n/Betsy’s Wedding` edits the name of the 2nd event from the results of the `efind` command to be `Betsy’s Wedding`
+* `eedit 3 dt/TA`  deletes the `TA` tag from the 3rd event.
+* `eedit 4 dt/*` deletes all tags from the 4th event.
+* `eedit 5 dt/Easy_exams t/Hard_exams` first deletes the `Easy_exams` tag, then adds `Hard_exams` tag to the 5th event.
+
+### Locating persons by name : `efind`
 
-### Clearing all entries : `clear`
+Finds events which contain any of the given keywords.
+
+Format: `efind KEYWORD [MORE_KEYWORDS]`
+
+* **Case-insensitive**. e.g `exams` will match `Exams`
+* Order of the keywords does not matter. e.g. `Exam Hard` will match `Hard Exam`
+* Only the **name** is searched.
+* Partial words can be matched e.g. `Exa` will match `CS2103T Exam` 
+* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Exam Hard` will return `Hard Exam`, `CS1101S Exams`
+
+Examples:
+`efind ex` returns `exams` and `Examinations`
+`efind CS Exam` returns `CS2100 Exam`,  `CS2101` 
+
+### Deleting a person : `edelete`
+
+Deletes the specified event from the event list.
+
+Format: `edelete INDEX1[-INDEX2]`
+
+* Deletes the event at the specified `INDEX1` or between the specified range from `INDEX1` to `INDEX2` inclusive.
+* `INDEX` refers to the index number shown in the displayed event list.
+* `INDEX` **must be a positive integer**, eg 1, 2, 3, …​
+
+Examples:
+`elist` followed by `edelete 2` deletes the 2nd event from the event list.
+`efind Exam` followed by `edelete 1` deletes the 1st event from the results of the `efind` command.
+`edelete 3-5` deletes events with index between 3 and 5 inclusively from the event list.
+
+
+### Clearing all events: `eclear`
+
+Clears all entries of all events from the event list.
```

Comment:
Should add a line for the format, even though it is obvious

Reply:
line added